scala-setup.sh

```diff
@@ -33,7 +37,7 @@ architecture() {
 
 architecture
 
-URL="${SCALA_CLI_BASE_URL}/scala-cli-x86_64-${OS_NAME}.gz"
+URL="${SCALA_CLI_BASE_URL}/scala-cli-${OS_ARCH}-${OS_NAME}.gz"
```

Comment:
I tested locally by echoing the url, I can confirm on my aarch64 machine the url will be correct now.